frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/utils.tsx

```diff
@@ -27,17 +27,16 @@ export type PodStatus = {
 };
 
 export const renderDetailItems = (details: DetailItem[]): React.ReactNode => (
-  <DescriptionList
-    isHorizontal
-    horizontalTermWidthModifier={{
-      default: '15ch',
-    }}
-  >
+  <DescriptionList isHorizontal horizontalTermWidthModifier={{ lg: '22ch', md: '15ch' }}>
     {details.map((detail) => (
       <DescriptionListGroup key={detail.key} data-testid={`detail-item-${detail.key}`}>
         <DescriptionListTerm>{detail.key}</DescriptionListTerm>
         <DescriptionListDescription data-testid="detail-item-value">
-          {detail.value}
+          {!detail.value && detail.value !== 0 ? (
+            <span className="pf-v5-u-disabled-color-100">No value</span>
```

Comment:
just want to confirm that `No value` is what we want. I think in some places we use `n/a` and in some places we use `-`

Reply:
I'm not sure if `n/a` makes sense here, but `-` might. I'm ok with either that or leave it as is. What do you think @yannnz?

Reply:
No value with gray color is the suggested consistency pattern for detail page. Hyphen is for table list.